.github/workflows/ci.yml

```diff
@@ -8,13 +8,37 @@ on:
     paths-ignore:
       - '**.md'
       - 'doc/**'
+      - '**.pdf'
+      - '**.tex'
+      - '**.jpg'
+      - '**.jpeg'
+      - '**.png'
+      - '**.bbl'
+      - '**.bib'
+      - 'doc/**.dat'
+      - 'doc/**.ipynb'
+      - 'doc/**.py'
+      - 'doc/**.sh'
+      - 'doc/**.xlsx'
   pull_request:
     branches:
       - master
       - develop
     paths-ignore:
       - '**.md'
       - 'doc/**'
+      - '**.pdf'
+      - '**.tex'
+      - '**.jpg'
+      - '**.jpeg'
+      - '**.png'
+      - '**.bbl'
+      - '**.bib'
+      - 'doc/**.dat'
+      - 'doc/**.ipynb'
+      - 'doc/**.py'
+      - 'doc/**.sh'
+      - 'doc/**.xlsx'
```

Comment:
What are these changes?

Reply:
@langevin-usgs, Initially, these PRs were not triggering the full suite of ci, these are some changes that @wpbonelli and I came up with so that PRs associated with adjustments to the documentation generally would not trigger the full suite of ci; however, if adjustments to a .dfn file are made, the thinking is that the full suite of ci should be run.  We tried coming up with a more reduced way of implementing this logic, but in the end it required what's shown.

Reply:
Okay, that's fine.  Just curious.